packages/manager/apps/pci-workflow/src/api/hooks/workflows.ts

```diff
@@ -0,0 +1,120 @@
+import { useQueries } from '@tanstack/react-query';
+import moment from 'moment';
+import { PaginationState } from '@ovhcloud/manager-components';
+import { applyFilters, Filter } from '@ovh-ux/manager-core-api';
+import { useRegions } from '@/api/hooks/regions';
+import {
+  getRegionsWorkflows,
+  TRemoteWorkflow,
+} from '@/api/data/region-workflow';
+import { getInstance } from '@/api/data/instance';
+
+export type TWorkflow = {
+  name: string;
+  id: string;
+  instanceId: string;
+  instanceName: string;
+  cron: string;
+  lastExecution: string;
+  lastExecutionStatus: string;
+  executions: TRemoteWorkflow['executions'];
+};
+
+export const paginateResults = (
+  items: unknown[],
+  pagination: PaginationState,
+) => ({
+  rows: items.slice(
+    pagination.pageIndex * pagination.pageSize,
+    (pagination.pageIndex + 1) * pagination.pageSize,
+  ),
+  pageCount: Math.ceil(items.length / pagination.pageSize),
+  totalRows: items.length,
+});
+
+export const useWorkflows = (projectId: string) => {
+  const { data: regions, isPending: isRegionsPending } = useRegions(projectId);
+
+  return useQueries({
+    queries: (regions || [])
+      .reduce((names: string[], { name, services }) => {
```

Comment:
it looks like it could be a filter instead of a reduce

Reply:
a copy/paste from existing code,didn't even try to look closer, will check